src/definitions/elements/segment.less

```diff
@@ -61,6 +61,7 @@
 --------------------*/
 
 /* Header */
+.ui.inverted.segment .sub.header,
```

Comment:
I think this should be 
```css
.ui.inverted.segment > .ui.header > .sub.header
```

Reply:
yes, it's better. fixed it accordingly

Reply:
Also changed fiddle in the description ;)